Specification.md

```diff
@@ -0,0 +1,87 @@
+- [Xdg Base Directories](https://www.freedesktop.org/wiki/Specifications/basedir-spec/)
+- [Xdg Desktop Entry](https://www.freedesktop.org/wiki/Specifications/desktop-entry-spec/)
+- [Xdg Desktop Menus](https://www.freedesktop.org/wiki/Specifications/menu-spec/)
+- [File Manager DBUS interface](https://www.freedesktop.org/wiki/Specifications/file-manager-interface/)
+- [File URI Specification](https://www.freedesktop.org/wiki/Specifications/file-uri-spec/)
+- [Free Media Player Specification](https://www.freedesktop.org/wiki/Specifications/free-media-player-specs/)
+- [Free Media Player Remote Interface Specification](https://www.freedesktop.org/wiki/Specifications/mpris-spec/)
+- [Xdg Icon Theme](https://www.freedesktop.org/wiki/Specifications/icon-theme-spec/)
+- [Xdg Icon Naming](https://www.freedesktop.org/wiki/Specifications/icon-naming-spec/)
+- [Shared mime Database](https://www.freedesktop.org/wiki/Specifications/shared-mime-info-spec/)
+- [Startup Notification](https://www.freedesktop.org/wiki/Specifications/startup-notification-spec/)
+- [Trashcan support](https://www.freedesktop.org/wiki/Specifications/trash-spec/)
+- [Bookmark Interchange Format](https://pyxml.sourceforge.net/topics/xbel/)
+
+TBD: Not sure on these:
+- [MIME Applications Associations](https://www.freedesktop.org/wiki/Specifications/mime-apps-spec/)
+- [XSettings](https://www.freedesktop.org/wiki/Specifications/xsettings-spec/)
+- [Cursor Conventions](https://www.freedesktop.org/wiki/Specifications/cursor-spec/)
+- [Sound Theme](https://www.freedesktop.org/wiki/Specifications/sound-theme-spec/)
+
+### Window decorations
+
+Applications which are part of the xapp project should support server side window decorations and use them by default. Xapps are free to optionally as well support client side decorations [^1].
+
+### Menus
+
+If menus are required for a xapps, traditional menu bars should be used by default. It is fine to optionally support e.g. hamburger menus [^2].
+
+TBD: Do we want to force a default on this?
+
+### Theming
+
+User theming has to be supported by xapps (TODO: Where is the spec for general theming?) [^3]
+
+### Desktop Environment Agnostic
+
+If the same windowing, theme and icon theme is used, XApps are intended to be visually consistent across Linux desktop environments. As such, they should not have dependencies to desktop-specific styles.
+
+### Identification as xapp application
+
+If an application fulfills all above criteria and is supposed to be added to the xapp project, it has to add a paragraph to a README.md file located on the top level of its git project with the following text:
+
+> ## Xapp compliance
+>
+> This application follows the standards defined in the [Xapp specification v1.0.0](insert link here) and as such will run fine on linux desktop environments which support freedesktop.org standards. The UI of this application will be well integrated on desktop environments which support the xapp standard.
```

Comment:
"will run fine" sounds kinda weird to me, and maybe leave out "Linux".  And calling out fd.o is maybe a little too restrictive; if we adopt/mandate standards outside of the fd.o umbrella, this will become out of date.  Overall, I think this required statement is a little too verbose.  I'm thinking of something simpler and shorter, like:

"This applications follows the standards defined in version x.y.z of the Xapp specification, and should integrate well into any desktop environment that has adopted the Xapp specification."

Reply:
Indeed, better to be more generic here, let's just use your suggestion: a852adcac09a3c86de2bb10c8e241a5add00f118